src/components/SearchMap/SearchMap.js

```diff
@@ -182,13 +181,13 @@ export class SearchMapComponent extends Component {
           activeListingId={activeListingId}
           mapComponentRefreshToken={this.state.mapReattachmentCount}
           createURLToListing={this.createURLToListing}
+          onClick={this.onMapClicked}
           onListingClicked={this.onListingClicked}
           onListingInfoCardClicked={this.onListingInfoCardClicked}
           onMapLoad={this.onMapLoadHandler}
-          onClick={this.onMapClicked}
           onMapMoveEnd={onMapMoveEnd}
-          zoom={zoom}
           reusableMapHiddenHandle={REUSABLE_MAP_HIDDEN_HANDLE}
+          zoom={zoom}
```

Comment:
Yeah, I was a bit hesitant to make this change, but my assumption is that most of the customizers have not touched these props - the need for changes is most likely happening inside those _SearchMapPriceLabel_, _SearchMapGroupLabel_ and _SearchMapInfoCard_ components. And this is a good moment to refactor these setups a bit since the props are changing anyway and they are going to be closer to each other.